master/buildbot/db/model.py

```diff
@@ -104,6 +104,7 @@ class Model(base.DBConnectorComponent):
     # claim is made by the master referenced by masterid.
     buildrequest_claims = sautils.Table(
         'buildrequest_claims', metadata,
+        sa.Column('dummy', sa.Integer, primary_key=True, autoincrement=True),
         sa.Column('brid', sa.Integer, sa.ForeignKey('buildrequests.id'),
```

Comment:
Still, that table is not necessary. Look this workflow:

1. The buildrequest is created in the buildrequest table, with the masterid column as NULL.
2. A message will be sent to all masters to inform them about it.
3. Each master capable of building this request will then try to update the masterid column only if is NULL, so other masters will not be able to update it anymore.